examples/build_twilio_go.py

```diff
@@ -3,6 +3,15 @@
 import re
```

Comment:
`Todo`: Rename file to `build_twilio_lib.py` to maintain consistency. 

Didn't update in this PR to avoid losing change history (Diff would show build_twilio_go as deleted, and build_twilio_lib file created).